pkg/glance/volumes.go

```diff
@@ -47,7 +47,13 @@ func GetVolumes(
 
 	for _, exv := range extraVol {
 		for _, vol := range exv.Propagate(svc) {
-			vm = append(vm, vol.Volumes...)
+			for _, v := range vol.Volumes {
+				convertedVolume := corev1.Volume{
+					Name:         v.Name,
+					VolumeSource: storage.ConvertVolumeSource(&v.VolumeSource),
```

Comment:
`pkg/glance/volumes.go:53:28: undefined: storage.ConvertVolumeSource`.
This iteration here should be fixed to be the same as manila [1]

[1] https://github.com/openstack-k8s-operators/manila-operator/pull/350/files#diff-c462f50b94934e03480506bb6bb04f55229f5092e655a12cef5c805d8c644a3eR62

Reply:
Rebased